codespell_lib/data/dictionary.txt

```diff
@@ -41496,6 +41504,8 @@ pytnon->python
 pytohn->python
 pyton->python
 pytyon->python
+qaulities->qualities
+qaulity->quality
```

Comment:
We already have a similar typo in the dictionary:
```
eqaulity->equality
```
If that's not asking too much, could you add the pural form of it?
```
eqaulities->equalities
```
And also:
```
ineqaulities->inequalities
ineqaulity->inequality
```